extensions/spring-web/deployment/src/main/java/io/quarkus/spring/web/deployment/ControllerAdviceAbstractExceptionMapperGenerator.java

```diff
@@ -27,12 +29,17 @@
 import io.quarkus.gizmo.MethodCreator;
 import io.quarkus.gizmo.MethodDescriptor;
 import io.quarkus.gizmo.ResultHandle;
+import io.quarkus.spring.web.runtime.ResponseContentTypeResolver;
 import io.quarkus.spring.web.runtime.ResponseEntityConverter;
 
 class ControllerAdviceAbstractExceptionMapperGenerator extends AbstractExceptionMapperGenerator {
 
     private static final DotName RESPONSE_ENTITY = DotName.createSimple("org.springframework.http.ResponseEntity");
-    private static final DotName STRING = DotName.createSimple(String.class.getName());
+
+    private static final List<String> TEXT_MEDIA_TYPES = Arrays.asList(
+            MediaType.TEXT_PLAIN, MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML, MediaType.TEXT_XML);
+    private static final List<String> OBJECT_MEDIA_TYPES = Arrays.asList(
```

Comment:
I am not sure I understand the distinction betweem `TEXT_MEDIA_TYPES` and `OBJECT_MEDIA_TYPES`. They seem to contains the same values?

Reply:
Aha, gotcha! Perhaps a comment in the code is in order? Because now it looks like a prime target for refactoring 😂